java/statestore/src/main/java/sleeper/statestore/s3/S3FileInfoStore.java

```diff
@@ -53,12 +51,15 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import static sleeper.statestore.s3.S3FileInfoFormat.createFileInfoSchema;
+import static sleeper.statestore.s3.S3FileInfoFormat.createFileReferenceCountSchema;
 import static sleeper.statestore.s3.S3RevisionUtils.RevisionId;
 import static sleeper.statestore.s3.S3StateStore.FIRST_REVISION;
 
 class S3FileInfoStore implements FileInfoStore {
```

Comment:
I'm not quite sure whether the changes to the S3 state store are worth doing this way, since we can achieve the same thing with the ready for GC records without having two files in S3. We could maybe avoid having more than one ready for GC record for each file, and only create it if there are no references left for that file.